ide/che-core-ide-ui/src/main/java/org/eclipse/che/ide/ui/smartTree/TreeNodeLoader.java

```diff
@@ -128,10 +128,11 @@ public void onLoad(final LoadEvent event) {
                 }
             }
 
-            //Set joint element on the view
-            Promise<Void> promise = Promises.resolve(null);
-            List<Node> exist = tree.getNodeStorage().getChildren(event.getRequestedNode());
-            chainChildrenAndSetJointElement(promise, exist.listIterator());
+            //Temporary disable it
```

Comment:
This method will be refactored with this issue https://jira.codenvycorp.com/browse/IDEX-3319
So to reduce requests count I temporary disable inner checking of childs to proper displayin ">" control of the node.